srcCxx/rti_connext_dds/ShapeType_configurator_rti_connext_dds.h

```diff
@@ -73,10 +74,15 @@ class ShapeTypeConfigurator {
         DDS_PropertyQosPolicyHelper_assert_property(&pQos.property,
                 "com.rti.serv.secure.access_control.governance_file",
                 governance_file, DDS_BOOLEAN_FALSE);
+
         DDS_PropertyQosPolicyHelper_assert_property(&pQos.property,
                 "com.rti.serv.secure.access_control.permissions_file",
                 permissions_file, DDS_BOOLEAN_FALSE);
 
+        DDS_PropertyQosPolicyHelper_assert_property(&pQos.property,
```

Comment:
Check return value.

Reply:
I added checks for the return value of all assertions